python/semantic_kernel/contents/streaming_chat_message_content.py

```diff
@@ -51,6 +53,12 @@ class StreamingChatMessageContent(ChatMessageContent, StreamingContentMixin):
         __add__: Combines two StreamingChatMessageContent instances.
     """
 
+    function_invoke_attempt: int | None = Field(
```

Comment:
I can see why this is needed more for streaming, but wouldn't regular CMC also benefit from this?

Reply:
I don't think the regular CMC will benefit from this because the regular CMCs aren't chunks. They contain the full content, and users won't need to concatenate them thus they don't need to know which request attempt a message belong to. The ordering of the CMC in the chat history already encodes that information.